### ISSUE TITLE
Failed to load the preview package

### ISSUE THREAD

**Issue by rgwch** [Glavin001/atom-preview#92]
[Enter steps to reproduce below:]

1. Just tried to install from the freshly installed Atom ->  Preferences -> Install
2. ...

**Atom Version**: 0.192.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: Mac OS X 10.10.3
**Thrown From**: [preview](https://github.com/Glavin001/atom-preview) package, v0.13.7

### Stack Trace

Failed to load the preview package

```
At Cannot read property 'prototype' of undefined

TypeError: Cannot read property 'prototype' of undefined
  at __extends (/Users/name/.atom/packages/preview/lib/preview-message-view.coffee:1:1)
  at /Users/name/.atom/packages/preview/lib/preview-message-view.coffee:4:3
  at Object.<anonymous> (/Users/name/.atom/packages/preview/lib/preview-message-view.coffee:28:5)
  at Object.<anonymous> (/Users/name/.atom/packages/preview/lib/preview-message-view.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Users/name/Applications/Atom.app/Contents/Resources/app/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/name/.atom/packages/preview/lib/preview-view.coffee:8:22)
  at Object.<anonymous> (/Users/name/.atom/packages/preview/lib/preview-view.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Users/name/Applications/Atom.app/Contents/Resources/app/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/name/.atom/packages/preview/lib/preview.coffee:2:15)
  at Object.<anonymous> (/Users/name/.atom/packages/preview/lib/preview.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Users/name/Applications/Atom.app/Contents/Resources/app/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Package.module.exports.Package.requireMainModule (/Users/name/Applications/Atom.app/Contents/Resources/app/src/package.js:667:34)
  at /Users/name/Applications/Atom.app/Contents/Resources/app/src/package.js:170:28
  at Package.module.exports.Package.measure (/Users/name/Applications/Atom.app/Contents/Resources/app/src/package.js:147:15)
  at Package.module.exports.Package.load (/Users/name/Applications/Atom.app/Contents/Resources/app/src/package.js:161:12)
  at PackageManager.module.exports.PackageManager.loadPackage (/Users/name/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:354:14)
  at PackageManager.module.exports.PackageManager.activatePackage (/Users/name/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:433:30)
  at /Users/name/Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:268:29
  at exit (/Users/name/Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:43:16)
  at BufferedProcess.triggerExitCallback (/Users/name/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:63:47)
  at ChildProcess.<anonymous> (/Users/name/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:85:18)
  at emitTwo (events.js:87:13)
  at ChildProcess.emit (events.js:169:7)
  at Process.ChildProcess._handle.onexit (child_process.js:1057:12)

```

### Commands

```
     -6:26.0 application:open (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -5:43.1 snippets:next-tab-stop (atom-text-editor.editor)
     -5:43.1 snippets:expand (atom-text-editor.editor)
  5x -5:41.9 core:move-down (atom-text-editor.editor)
     -5:36.4 core:save (atom-text-editor.editor)
     -5:06.2 timecop:view (atom-text-editor.editor)
  4x -4:40.7 core:backspace (atom-text-editor.editor.mini)
     -3:56.4 core:confirm (atom-text-editor.editor.mini)
     -1:03.4 command-palette:toggle (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
  4x -0:52.9 core:backspace (atom-text-editor.editor.mini)
     -0:50.5 editor:consolidate-selections (atom-text-editor.editor.mini)
     -0:50.5 core:cancel (atom-text-editor.editor.mini)
     -0:49.3 tool-panel:unfocus (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
language-jade, v0.4.0
preview, v0.13.7

# Dev
No dev packages
```

**Comment by melitta** [comment]
+1

**Comment by Glavin001** [comment]
Can you both provide me with some more information. I am unable to reproduce this error.

**Comment by Hunter-Dolan** [comment]
@Glavin001 This issue just appeared after I upgraded Atom to 0.198.0. The error message pops up right away once you launch. It seems this update broke a number of packages.

I'm working on a fix for this package along with several others. I'll make a pull request and reference this issue if manage to fix it.

**Comment by Glavin001** [comment]
I am also on Atom v0.198.0 and I just published v0.14.0 that fixes all deprecation warnings ( #94 ). Let me know if this is still an issue with v0.14.0
